apps/site/src/views/Home/sections/FAQ/FAQ.tsx

```diff
@@ -1,11 +1,31 @@
-import Container from "react-bootstrap/Container";
+import { getQuestions } from "./getQuestions";
+import FAQAccordion from "./FAQAccordion";
+
+import Image from "next/image";
+import { PortableText } from "@portabletext/react";
+import styles from "./FAQ.module.scss";
+
+import star from "@/assets/images/star.png";
+import eraser from "@/assets/images/eraser.png";
+
+export default async function FAQ() {
+	const questions = await getQuestions();
+	const faq = questions[0]["faqs"].map(({ _key, question, answer }) => ({
```

Comment:
Thought (non-blocking): the indexing and property accessor might make more sense as being part of `getQuestions`.

Reply:
I think this could go either way honestly. Since we're not really reusing the FAQ items, I'll just leave it as is for now.